### PR TITLE
Drop router and OpenWrt code.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

File: static/images/wifi-secure.svg [hunk 1]
```diff
@@ -1,49 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (benfrancis) [inline static/images/wifi-secure.svg:1]: Is this image not still used in Wi-Fi setup? https://github.com/WebThingsIO/gateway/blob/master/src/wifi-setup.js#L136

File: static/css/lib/highlight-9.12.0.min.css [hunk 2]
```diff
@@ -1 +0,0 @@
-.hljs{display:block;overflow-x:auto;padding:0.5em;background:#F0F0F0}.hljs,.hljs-subst{color:#444}.hljs-comment{color:#888888}.hljs-keyword,.hljs-attribute,.hljs-selector-tag,.hljs-meta-keyword,.hljs-doctag,.hljs-name{font-weight:bold}.hljs-type,.hljs-string,.hljs-number,.hljs-selector-id,.hljs-selector-class,.hljs-quote,.hljs-template-tag,.hljs-deletion{color:#880000}.hljs-title,.hljs-section{color:#880000;font-weight:bold}.hljs-regexp,.hljs-symbol,.hljs-variable,.hljs-template-variable,.hljs-link,.hljs-selector-attr,.hljs-selector-pseudo{color:#BC6060}.hljs-literal{color:#78A960}.hljs-built_in,.hljs-bullet,.hljs-code,.hljs-addition{color:#397300}.hljs-meta{color:#1f7199}.hljs-meta-string{color:#4d99bf}.hljs-emphasis{font-style:italic}.hljs-strong{font-weight:bold}
```
Comment (benfrancis) [inline static/css/lib/highlight-9.12.0.min.css:1]: Is this not still used by the local token developer settings UI? https://github.com/WebThingsIO/gateway/blob/master/src/views/local-token-service.handlebars#L14

File: static/things.json [hunk 3]
```diff
@@ -1,3 +0,0 @@
-[
-
-]
```
Comment (benfrancis) [inline static/things.json:3]: LOL, what's that still doing there? Oh, I left it there 😂

**Review by benfrancis — CHANGES_REQUESTED** [review]

File: static/images/wifi-secure.svg (hunk 1 above)
Comment (mrstegeman) [reply static/images/wifi-secure.svg]: Good catch! I was looking in the wrong place.

File: static/things.json (hunk 3 above)
Comment (mrstegeman) [reply static/things.json:3]: Haha I don't know how I've completely missed that file for over 3 years.

File: static/css/lib/highlight-9.12.0.min.css (hunk 2 above)
Comment (mrstegeman) [reply static/css/lib/highlight-9.12.0.min.css:1]: Good catch! We were using a very outdated one, too, so I've updated it.

**Review by benfrancis — APPROVED** [review]

**Pull request merged**